### DISCUSSION TITLE
Unique names of PDF reports

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#63] [Shipped - Live on production]
Some PDF reports have only generic names. If you export several at once, it is impossible to see directly which report belongs to which device, person or tenant.

For example, the report "Medical products book". The file name is simply "Medical book". 
The device number in the file name would already help a lot.

The same applies to the report "Employe trainings". The file name is simply "staff".

**Comment by YvesR** [comment]
And how do you name it then if you create n-devices in 1 report?

**Reply by moritzvolkmar** [reply]
Good point. Are different names possible for the same report?
Ex.: 
1 device selected = report name + device number (Medical device book 1234)
n devices selected = report name (Medical device books)
